cmd/clusterlink/cmd/deploy/deploy_peer.go

```diff
@@ -103,6 +116,14 @@ func (o *PeerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.Uint16Var(&o.IngressPort, "ingress-port", apis.DefaultExternalPort,
 		"Represents the ingress port. By default it is set to 443 for LoadBalancer"+
 			" and a random port in range (30000 to 32767) for NodePort.\nThis option is only valid if --autostart is set.")
+	fs.StringVar(&o.DataplaneType, "dataplane", platform.DataplaneTypeEnvoy,
+		"Type of dataplane, Supported values: \"envoy\" (default), \"go\"")
+	fs.Uint16Var(&o.DataplaneReplicas, "dataplane-replicas", 1, "Number of dataplanes.")
+	fs.StringVar(&o.LogLevel, "log-level", "info",
+		"The log level. One of fatal, error, warn, info, debug.")
+	fs.BoolVar(&o.DryRun, "dry-run", false,
+		"Dry-run doesn't deploy the ClusterLink operator but creates the ClusterLink instance YAML")
+	fs.BoolVar(&o.CRDMode, "crd-mode", false, "Run a CRD-based controlplane.")
```

Comment:
still needed? when can it be removed?

Reply:
Regardless, shouldn't CRD be defaulted to `true`?

Reply:
Actually is not really needed for the operator, because it supports only CRD mode. It is for the k8s.yaml and in the next version we will remove it.